docs/components/skeleton.md

```diff
@@ -126,23 +126,62 @@ This loading state is used for a spinner, which can be used for a wide variety o
 
 The full screen loading state is used to display a full screen loader typically during initial render of an app to avoid any FOUC (Flash Of Unstyled Content) while the app tries to figure out if you are able to access the route and also to perform any expensive querying on first load.
 
+### progress
+
+Used for controlling the progress indicator.
+
+### hideProgress
+
+Defaults to `false`, you can use this prop to hide the progress indicator.
+
 <div>
-  <KButton @click="clicked()" class="mr-2">click for default progress behavior</KButton>
-  <KButton @click="clickProgress()">click me to simulate progress manually</KButton>
+  <KButton
+  class="mr-2 mb-2" @click="clickNoProgress()">Click for no progress indicator</KButton>
+  <KButton class="mr-2 mb-2" @click="clicked()">Click for default progress behavior</KButton>
+  <KButton @click="clickProgress()">Click me to simulate progress manually</KButton>
+  <KSkeleton
+    v-if="loadingNone"
+    :delay-milliseconds="0"
+    hide-progress
+    type="fullscreen-kong"
+  />
   <KSkeleton
     v-if="loading"
+    :delay-milliseconds="0"
     type="fullscreen-kong"
-    :delay-milliseconds="0" />
+  />
   <KSkeleton
     v-if="loadingManually"
-    type="fullscreen-kong"
+    :delay-milliseconds="0"
     :progress="progress"
-    :delay-milliseconds="0" />
+    type="fullscreen-kong"
+  />
 </div>
 
 ```html
-  <KButton @click="clicked()">click for default progress behavior</KButton>
-  <KSkeleton v-if="loading" type="fullscreen-kong" :delay-milliseconds="0" />
+  <KButton @click="clickNoProgress()">Click for no progress indicator</KButton>
+  <KButton @click="clicked()" class="mr-2">Click for default progress behavior</KButton>
+  <KButton @click="clickProgress()">Click me to simulate progress manually</KButton>
+
+  <KSkeleton
+    v-if="loadingNone"
+    :delay-milliseconds="0"
+    hide-progress
+    type="fullscreen-kong"
+  />
+
+  <KSkeleton
+    v-if="loading"
+    :delay-milliseconds="0"
+    type="fullscreen-kong"
+  />
+
+  <KSkeleton
+    v-if="loadingManually"
+    :delay-milliseconds="0"
+    :progress="progress"
+    type="fullscreen-kong"
+  />
```

Comment:
suggestion: Just for this example, can we show how we are manually doing the progress in the example? Might be helpful